blog/2015-03-31-Eclipse_Docker_Tooling.adoc

```diff
@@ -0,0 +1,90 @@
+= Upcoming Docker Tooling for Eclipse
+:page-layout: blog
+:page-author: xcoulon
+:page-tags: [jbosstools, docker, eclipse]
+
+We are currently busy working on Docker tooling for Eclipse and even though the
+code is not public yet (more details on that at the end of this blog entry), we thought it
+would be interesting to share our progress on the project.
+
+The Docker tooling is aimed at providing in Eclipse the same level features as
+users have with the command-line interface.
+We rely on the https://github.com/spotify/docker-client[open source Docker client
```

Comment:
I would put that at the end near "where is the code" or just drop it since its a detail so far for this blog.